packages/apps/desktop-viewer-test/src/frontend/components/routes/ViewerRoute.tsx

```diff
@@ -35,10 +38,14 @@ export const ViewerRoute = ({ location }: ViewerRouteProps) => {
     <Viewer
       filePath={filePath}
       onIModelAppInit={onIModelAppInitialized}
-      defaultUiConfig={{
-        contentManipulationTools: { cornerItem: { hideDefault: true } },
-      }}
```

Comment:
Is contentManipulationTools deprecated? The type of defaultUIConfig is from ItwinViewerUi
![image](https://user-images.githubusercontent.com/51834103/173899301-8efa8e6e-ae3d-472d-8b4e-cac5b5629cb6.png)

Reply:
Should the desktop viewer look exactly like the web viewer? When I ran the desktop viewer, it didn't have any navigation tools either:
![image](https://user-images.githubusercontent.com/51834103/173901153-bbd77102-ebbd-497f-8de4-bf39428b4521.png)

Reply:
Oh yeah, my bad, I haven't done git pull in a long time, that was why.

Reply:
looks like you dropped your changes to the desktop side with the merge.

make sure you did:
```
git fetch --all
git merge origin/next
```
then resolve any conflicts.
look at the diff in your pr here to see what changes were made